tldraw/packages/core/src/lib/tools/TLSelectTool/states/IdleState.ts

```diff
@@ -47,7 +47,8 @@ export class IdleState<
     } = this.app
 
     if (event.button === 2) {
-      this.tool.transition('contextMenu', info)
+      // relies on @radix-ui/react-context-menu Root's onOpenChange now
+      // this.tool.transition('contextMenu', info)
```

Comment:
Also related to https://github.com/logseq/logseq/pull/7282#discussion_r1018378532

Reply:
This is reverted to support the previous behavior